### ISSUE TITLE
How can I load a DLL ?

### ISSUE THREAD

**Issue by natinusala** [Ancurio/mkxp#93]
Hello,

Since the Win32API is not working yet, is there a way I could load a DLL ? I would like to try the Neo Mode 7 with mkxp.

Thanks !

**Comment by cremno** [comment]
That isn't possible, since it relies on internals of the original RGSS and mkxp doesn't replicate them.

**Comment by natinusala** [comment]
Do you plan to make it or absolutely not ?

**Comment by cremno** [comment]
I can only think of a way that would require small(-ish) edits to such scripts. Of course, using DLLs prevents your game from running natively on other OS than Windows, which is one of the major benefits of mkxp.

The most preferable way is modifying mkxp's source code, because then you (likely) have better performance and it also supports other OS. I think @Ancurio was working on (or something similar to) Mode 7, if I remember correctly.

**Comment by natinusala** [comment]
The thing is that I don't have the source code of the DLL I would like to use, should I contact the author of the script and DLL so he can port it to mkxp ?

**Comment by tarceri** [comment]
There are Mode 7 scripts that don't require the use of an external dll. You could try one of them instead.

**Comment by natinusala** [comment]
I can't find any (for RMXP, it's important), only NeoMode7 and H-Mode7 which both requires a DLL

**Comment by tarceri** [comment]
I can't remember where I found these so I zipped them up and put them on my website. I also think I have tweaked them a tiny bit to get them to work on the version of Ruby MKXP uses, can't remember as it was a while ago I got them working and haven't actually used them in my game yet.

They are written by the same person that wrote the other ones but are in pure Ruby. I guess maybe they wanted more speed so moved to using dll's but on modern hardware it shouldn't really be much of an issue anymore.

http://www.itsqueeze.com/wp-content/uploads/2015/02/Mode7.tar.gz
